README.md

```diff
@@ -8,16 +8,23 @@ Variant Normalization relies on some local data caches which you will need to se
 From the _variant_ directory of the repository:
 ```
 pipenv sync
-curl ftp://ftp.ebi.ac.uk/pub/databases/genenames/new/tsv/non_alt_loci_set.txt > data/gene_symbols.txt
+curl ftp://ftp.ebi.ac.uk/pub/databases/genenames/new/tsv/hgnc_complete_set.txt.txt > data/gene_symbols.txt
 pip install seqrepo
 mkdir -p data/seqrepo
-seqrepo -r data/seqrepo pull -i 2020-11-27
-chmod -R u+w data/seqrepo
+seqrepo -r data/seqrepo pull -i 2021-01-29
+sudo chmod -R u+w data/seqrepo
 cd data/seqrepo
 seqrepo_date_dir=$(ls -d */)
-mv $seqrepo_date_dir latest
+sudo mv $seqrepo_date_dir 2021-01-29
```

Comment:
remind me to make any necessary README changes to metakb issue-59 if we ever get around to reviewing it

Reply:
I changed it back to latest